### ISSUE TITLE
VirusTotal shows virus in kubelogin.exe (v1.25.3)

### ISSUE THREAD

**Issue by slog2000** [int128/kubelogin#821]
## Describe the question
I've checked kubelogin.exe (v1.25.3) from kubelogin_windows_amd64.zip at the VirusTotal.com. It shows: Trojan.Malware.300983.susgen.
You can see the result: https://www.virustotal.com/gui/file/b233c819613aa710ce1c41eefbc62514796013c6d64f47ceb27f2fdbbebb7cff
I also checked previous versions v1.25.2 and v1.25.1: they also have virus.

Of course, it can be "false positive", but can you check and build windows release without "red flags"?

## To reproduce
1. Open in the browser: https://www.virustotal.com/gui/home/upload
2. Click "Choose file"
3. Select "kubelogin.exe"

## Your environment
- OS: Windows 10
- kubelogin version: v1.25.3
- kubectl version: v1.24.0
- OpenID Connect provider: keycloak

**Comment by int128** [comment]
This is false positive, because all binaries are built from this repository by GitHub Actions.

I'd like to fix this issue but could not find a way by googling Trojan.Malware.300983.susgen. Anyone knows it?

**Comment by slog2000** [comment]
Thank you for your answer.
Go lang support says: https://go.dev/doc/faq#virus
